packages/contracts-bedrock/snapshots/storageLayout/L1StandardBridge.json

```diff
@@ -36,37 +22,58 @@
   },
   {
     "bytes": "20",
-    "label": "messenger",
+    "label": "spacer_3_0_20",
     "offset": 0,
     "slot": "3",
-    "type": "contract ICrossDomainMessenger"
+    "type": "address"
   },
   {
     "bytes": "20",
-    "label": "otherBridge",
+    "label": "spacer_4_0_20",
     "offset": 0,
     "slot": "4",
-    "type": "contract StandardBridge"
+    "type": "address"
   },
   {
-    "bytes": "1440",
+    "bytes": "1408",
     "label": "__gap",
     "offset": 0,
     "slot": "5",
-    "type": "uint256[45]"
+    "type": "uint256[44]"
+  },
+  {
+    "bytes": "1",
+    "label": "_initialized",
+    "offset": 0,
+    "slot": "49",
+    "type": "uint8"
+  },
+  {
+    "bytes": "1",
+    "label": "_initializing",
+    "offset": 1,
+    "slot": "49",
+    "type": "bool"
   },
   {
     "bytes": "20",
     "label": "superchainConfig",
-    "offset": 0,
-    "slot": "50",
+    "offset": 2,
+    "slot": "49",
```

Comment:
Need a spacer above to prevent packing the `superchainConfig` into the `_initialized` slot.